pipenv/cli/command.py

```diff
@@ -762,18 +762,24 @@ def requirements(state, dev=False, dev_only=False, hash=False):
         echo(crayons.normal(" ".join([prefix, package_index["url"]])))
     if not dev_only:
         for req_name, value in lockfile["default"].items():
-            if hash:
-                hashes = [f" \\\n    --hash={h}" for h in value.get("hashes", [])]
+            if value.get("editable", False):
+                echo(crayons.normal("-e " + value["path"]))
             else:
```

Comment:
You don't need to use `crayons.normal` inside `echo`.

Reply:
Oh, I kept previous source codes,
I will remove `crayons.normal` without converting into `click`,
Thanks.